src/Support/Part.php

```diff
@@ -0,0 +1,392 @@
+<?php
+
+namespace Laravel\Vapor\Support;
+
+use InvalidArgumentException;
+use LogicException;
+
+/**
+ * @see https://github.com/Riverline/multipart-parser/pull/41
+ */
+class Part
```

Comment:
Because I predict that the `riverline/multipart-parser` package is no longer maintained, I've copied the class for now.